<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Diary</class>
 <widget class="QMainWindow" name="Diary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>日记编辑软件</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>911</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="rb_background_music">
       <property name="text">
        <string>背景音乐</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_time">
       <property name="text">
        <string>现在时间</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="pte_diary">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>911</width>
      <height>341</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正字迹-海体正楷 简</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="plainText">
     <string>请输入内容</string>
    </property>
    <property name="placeholderText">
     <string>请输入内容</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正字迹-海体正楷 简</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string>日记编辑器</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>十字兄弟有限公司出品</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>520</y>
      <width>911</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pb_choose_file">
       <property name="text">
        <string>选择文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_finish">
       <property name="text">
        <string>编辑完成</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1047</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
